test/images/OWNERS

```diff
@@ -0,0 +1,7 @@
+reviewers:
+  - luxas
+  - mkumatag
+  - ixdy
+approvers:
+  - luxas
+  - ixdy
```

Comment:
this is redundant since I'm already in `test/OWNERS`, but being explicit is also fine.